content/en/blog/_posts/2020-12-10-third-party-device-metrics-hits-ga.md

```diff
@@ -0,0 +1,134 @@
+---
+ layout: blog
+ title: 'Third Party Device Metrics Hits G.A'
+ date: 2020-12-10
+ slug: third-party-device-metrics-hits-ga
+ ---
+
+ **Authors:** Renaud Gaubert, David Ashpole, and Pramod Ramarao
+
+ With Kubernetes 1.20, infrastructure teams managing large scale Kubernetes clusters, are seeing the graduation of two exciting and long time coming features:
+ * The Pod Resources API, introduced in 1.13 is finally graduating to GA. This allows Kubernetes plugins to obtain information about the node’s resource usage and assignment (e.g: which pod/container consumes which device)
+ * The DisableAcceleratorMetrics feature, introduced in 1.19 is graduating to beta (and will be enabled by default). This removes device metrics reported by the kubelet in favor of the new plugin architecture.
+
+Many of the features related to fundamental device support (device discovery, plugin, and monitoring) are reaching a strong level of stability.
+Kubernetes users should see these features as stepping stones to enable more  complex use cases (networking, scheduling, storage, etc.)!
+
+One such example is Non Uniform Memory Access (NUMA) placement, where when selecting a device, an application typically wants to ensure that data transfer between CPU Memory and Device Memory is as fast as possible. In some cases, incorrect NUMA placement can nullify the benefit of offloading compute to an external device.
+
+If these are topics you are interested in or feel strongly about, consider joining `sig-node` for all topics related to the kubernetes node, the COD (container orchestrated device) workgroup for topics related to runtimes, or the resource management forum for  topics related to resource management!
+
+# The Pod Resources API - Why does it need to exist?
+
+Kubernetes is a vendor neutral platform, which typically has long release cycles. If we want Kubernetes to support device monitoring, adding vendor-specific code in the kubernetes code base is not an ideal solution. Ultimately, devices are a domain where deep expertise is needed and the best people to add and maintain code in that area are the device vendors themselves.
```

Comment:
I think it's a question of perspective :D 

From a vendor perspective it can take 1-2 years before a new feature (or a device support) hits customers. 
In any case, I'm trying to express that a vendor has more control on the release timeline with a plugin. Not that the kubernetes release cycle is too slow (I like it that way).

So we can remove that statement :D !